client/my-sites/customer-home/cards/notices/celebrate-notice-v2/index.jsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { connect, useDispatch } from 'react-redux';
+import { Button } from '@automattic/components';
+import { isDesktop } from '@automattic/viewport';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
+import { requestHomeLayout } from 'state/home/actions';
+import { savePreference } from 'state/preferences/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Image dependencies
+ */
+import fireworksIllustration from 'assets/images/customer-home/illustration--fireworks-v2.svg';
+
+const CelebrateNotice = ( {
+	actionText,
+	description,
+	noticeId,
+	illustration = fireworksIllustration,
+	onSkip,
+	showSkip = false,
+	skipText,
+	siteId,
+	title,
+} ) => {
+	const [ isLoading, setIsLoading ] = useState( false );
+	const [ isVisible, setIsVisible ] = useState( true );
+	const dispatch = useDispatch();
+
+	if ( ! isVisible ) {
+		return null;
+	}
+
+	const dismissalPreferenceKey = `dismissible-card-home-notice-${ noticeId }-${ siteId }`;
+
+	const showNextTask = () => {
+		setIsLoading( true );
+		dispatch( savePreference( dismissalPreferenceKey, true ) );
```

Comment:
So here's the race condition, we can request the next layout fetch before this is updated on the backend.

Reply:
I can fix this on the client, but in the future though it'd be more robust to create another POST endpoint to do both parts in one request. Maybe something off of `/home/skip-task`